test/XRP/web-socket-network-client.test.ts

```diff
@@ -176,4 +181,89 @@ describe('WebSocket Tests', function (): void {
       }
     }
   })
+
+  it('getAccountOffers - valid requests', async function (): Promise<void> {
```

Comment:
why are we using the `function` keyword instead of arrow functions?

```javascript
it('getAccountOffers - valid requests', async (): Promise<void> => {
   ...
})
```

Reply:
I think that's how all the other test files do it (and maybe eslint gets mad at it?)